tests/unit/model/test_model.py

```diff
@@ -293,6 +296,40 @@ def test_feature_columns(object_under_test):
     assert object_under_test.feature_columns == expected
 
 
+def test_transform_columns():
+    from google.cloud.bigquery import standard_sql
+    input_resource_ = {
+        "name": "is_male",
+    "type": {
+        "typeKind": "BOOL"
+    },
+    "transformSql": "is_male"
+    }
+    google.cloud.bigquery.model.TransformColumn(input_resource_)
+
+def test_transform_column_name():
+    transform_column = google.cloud.bigquery.model.TransformColumn({"name":"is_female"})
+    assert transform_column.name == "is_female"
+
+def test_transform_column_transform_sql():
+    transform_column = google.cloud.bigquery.model.TransformColumn({"transformSql": "is_female"})
+    assert transform_column.transform_sql == "is_female"
+
+def test_transform_column_type():
+    transform_column = google.cloud.bigquery.model.TransformColumn({"type":{"typeKind": "BOOL"}})
+    assert transform_column.type_.type_kind == "BOOL"
+
+def test_transform_column_type_none():
+    transform_column = google.cloud.bigquery.model.TransformColumn({"type":None})
+    assert transform_column.type_ is None
+
+def test_transform_column_properties():
+    transform_column = google.cloud.bigquery.model.TransformColumn({"name": str,"type": {
```

Comment:
Please use real string/bool objects as the values in your test dictionary. Passing through function objects is not realistic.

See: Way (1) in https://learnpython.com/blog/create-dictionary-in-python/